### PR TITLE
Feature: me

### PR DESCRIPTION
**Pull request by meltingscales**
the feature is just me. 🥺

### CODE REVIEW

**Comment by supersat** [comment]
Your PR is #19, but it renders as [9](https://github.com/endingwithali/boyfriendlist/blob/7d621e0b32908575b276e01fe89494b6818b0d11/README.md). You should revise.

**Comment by endingwithali** [comment]
conflicts pls fix

**Comment by meltingscales** [comment]
Fixed! Merge pls?

File: README.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ Submit a PR with a link to your twitch or github handle to join the boyfriend li
   <li><a href="https://github.com/supersat">supersat</a></li>
   <li><a href="https://github.com/dmfj">dmfj</a></li>
   <li><a href="https://github.com/data-diddler">data-diddler</a></li>
+  <li><a></a></li>
+  <li><a></a></li>
+  <li><a></a></li>
+  <li><a></a></li>
+  <li><a></a></li>
```
Comment (igorbrigadir) [inline README.md:23]: Remove extra blank entries

**Review by igorbrigadir — CHANGES_REQUESTED** [review]
Remove extra blank entries

**Pull request merged**